src/elements/InputField/InputField.vue

```diff
@@ -77,7 +78,9 @@
       Simplified,
 
       Invertible,
-      Unactionable
+      Unactionable,
+      Raisable
```

Comment:
@chidexebere why have you added `Raisable` here?

Reply:
Apologies , in the previous design, sides and bottom shadows were included. With the updated design, it was removed. I was meant to remove it from the component but I only removed it from the story.

Reply:
Oh, okay. Cool :+1: